src/PlatformItemGroups.props

```
@@ -1,76 +1,79 @@
-    <IncludedPlatformExtension Condition="$(_IsNetStd) and '$(UnoRuntimeIdentifier)'=='WebAssembly'" Include="wasm" />
-    <IncludedPlatformExtension Condition="$(_IsNetStd) and '$(UnoRuntimeIdentifier)'=='Skia'"  Include="skia" />
-    <IncludedPlatformExtension Condition="$(_IsNetStd) and '$(UnoRuntimeIdentifier)'=='Reference'" Include="netstdref" />
-    <IncludedPlatformExtension Condition="$(_IsIOS)" Include="iOS" />
-    <IncludedPlatformExtension Condition="$(_IsMacOS)" Include="macOS" />
-    <IncludedPlatformExtension Condition="$(_IsMacOS) or $(_IsIOS)" Include="iOSmacOS" />
-    <IncludedPlatformExtension Condition="$(_IsAndroid)" Include="Android" />
-    <IncludedPlatformExtension Condition="$(_IsAndroid) or $(_IsIOS) or $(_IsMacOS)"  Include="Xamarin" />
-    <IncludedPlatformExtension Condition="'$(TargetFramework)'=='uap10.0'" Include="WinRT" />
-    <IncludedPlatformExtension Condition="'$(TargetFramework)'=='net461'" Include="net" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <AndroidResource Include="Resources\**\*.xml" Condition="$(_IsAndroid)" />
-
-    <None Include="**\*.%(IncludedPlatformExtension.Identity).cs" Exclude="bin\**\*.%(IncludedPlatformExtension.Identity).cs;obj\**\*.%(IncludedPlatformExtension.Identity).cs" />
-    <Compile Remove="**\*.%(IncludedPlatformExtension.Identity).cs" />
-    <Compile Include="**\*.%(IncludedPlatformExtension.Identity).cs"
-             Exclude="bin\**\*.%(IncludedPlatformExtension.Identity).cs;obj\**\*.%(IncludedPlatformExtension.Identity).cs" />
-
-    <Page Include="**\*.xaml"
-          Exclude="bin/**/*.xaml;obj/**/*.xaml"
-          Condition="$(_IsAndroid) or $(_IsIOS) or $(_IsUWP) or $(_IsNetStd) or $(_IsMacOS)" />
-
-  </ItemGroup>
+	<PropertyGroup>
+		<_IsAndroid>false</_IsAndroid>
+		<_IsAndroid Condition="$(IsMonoAndroid) or '$(TargetFramework)'=='net6.0-android'">true</_IsAndroid>
+
+		<_IsCatalyst>false</_IsCatalyst>
+		<_IsCatalyst Condition="$(_IsCatalyst) or '$(TargetFramework)'=='net6.0-maccatalyst'">true</_IsCatalyst>
+
+		<_IsIOS>false</_IsIOS>
+		<_IsIOS Condition="$(IsXamarinIOS) or '$(TargetFramework)'=='net6.0-ios' or '$(TargetFramework)'=='net6.0-maccatalyst'">true</_IsIOS>
+
+		<_IsMacOS>false</_IsMacOS>
+		<_IsMacOS Condition="'$(TargetFramework)'=='xamarinmac20' or '$(TargetFramework)'=='net6.0-macos'">true</_IsMacOS>
+
+		<_IsNet>false</_IsNet>
+		<_IsNet Condition="'$(TargetFramework)'=='net461'">true</_IsNet>
+
+		<_IsNetCore>false</_IsNetCore>
+		<_IsNetCore Condition="'$(TargetFramework)'=='net5.0' or '$(TargetFramework)'=='net6.0' or '$(TargetFramework)'=='net7.0'">true</_IsNetCore>
+
+		<_IsNetStd>false</_IsNetStd>
+		<_IsNetStd Condition="'$(TargetFramework)'=='netstandard2.0' or $(_IsNetCore)">true</_IsNetStd>
+
+		<_IsUWP>false</_IsUWP>
+		<_IsUWP Condition="'$(TargetFramework)'=='uap10.0'">true</_IsUWP>
+
+		<_IsWinUI>false</_IsWinUI>
+		<_IsWinUI Condition="$(TargetFramework.Contains('windows10'))">true</_IsWinUI>
```

It's probably best to use this instead (Since `TargetPlatformIdentifier` cannot always be used):
```suggestion
		<_IsWinUI Condition="$(TargetFramework.Contains('-windows'))">true</_IsWinUI>
```